src/collections/Form/Fields.js

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+import React, {Children, Component, PropTypes} from 'react';
+import classNames from 'classnames';
+import getUnhandledProps from 'src/utils/getUnhandledProps';
+import numberToWord from 'src/utils/numberToWord';
+import META from 'src/utils/Meta.js';
+
+export default class Fields extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    /**
+     * Dynamically adds className='<count> fields'.
+     */
+    evenlyDivided: PropTypes.bool,
+  };
+
+  static _meta = {
+    library: META.library.semanticUI,
+    name: 'Fields',
+    parent: 'Form',
+    type: META.type.collection,
+  };
+
+  render() {
+    let fieldCount = 0;
+    Children.forEach(this.props.children, child => {
+      _.get(child, 'type._meta.name') === 'Field' && fieldCount++;
+    });
+    fieldCount = numberToWord(fieldCount);
+
+    const classes = classNames(
+      'sd-fields',
+      this.props.evenlyDivided && fieldCount,
```

Comment:
Using the && syntax here as the would be object key name is a variable, `fieldCount`.